ichnaea/locate/searcher.py

```diff
@@ -118,6 +120,9 @@ class PositionSearcher(Searcher):
         ('wifi', (
             WifiPositionProvider,
         )),
+        ('fallback', (
+            FallbackProvider,
+        )),
```

Comment:
Ok, then ignore this. The weak check is actually a sensible one. We'll probably end up changing this bit soon enough. Maybe something like "if fallback is allowed for this request, do that first", so it's going to be more of a preferred provider, or do both lookups in our own data and the fallback provider and collect some metrics around how those two results differ from each other. Not sure yet, so leave as is for now :)